internal/log/logger/type_test.go

```diff
@@ -23,200 +24,243 @@ import (
 	"go.uber.org/goleak"
 )
 
-func TestString(t *testing.T) {
+func TestType_String(t *testing.T) {
+	type want struct {
+		want string
+	}
 	type test struct {
-		name    string
-		logType Type
-		want    string
+		name       string
+		m          Type
+		want       want
+		checkFunc  func(want, string) error
+		beforeFunc func()
+		afterFunc  func()
+	}
+	defaultCheckFunc := func(w want, got string) error {
+		if !reflect.DeepEqual(got, w.want) {
+			return errors.Errorf("got = %v, want %v", got, w.want)
+		}
+		return nil
 	}
-
 	tests := []test{
 		{
-			name:    "returns glg",
-			logType: GLG,
-			want:    "glg",
+			name: "returns glg when m is GLG",
+			m:    GLG,
+			want: want{
+				want: "glg",
+			},
 		},
 
 		{
-			name:    "returns zap",
-			logType: ZAP,
-			want:    "zap",
+			name: "returns zap when m is ZAP",
+			m:    ZAP,
+			want: want{
+				want: "zap",
+			},
 		},
 
 		{
-			name:    "returns zerolog",
-			logType: ZEROLOG,
-			want:    "zerolog",
+			name: "returns zerolog when m is ZEROLOG",
+			m:    ZEROLOG,
+			want: want{
+				want: "zerolog",
+			},
 		},
 
 		{
-			name:    "returns logrus",
-			logType: LOGRUS,
-			want:    "logrus",
+			name: "returns logrus when m is LOGRUS",
+			m:    LOGRUS,
+			want: want{
+				want: "logrus",
+			},
 		},
 
 		{
-			name:    "returns klog",
-			logType: KLOG,
-			want:    "klog",
+			name: "returns klog when m is KLOG",
+			m:    KLOG,
+			want: want{
+				want: "klog",
+			},
 		},
 
 		{
-			name:    "returns unknown",
-			logType: Type(100),
-			want:    "unknown",
+			name: "returns unknown when m is unknown",
+			m:    Type(100),
+			want: want{
+				want: "unknown",
+			},
 		},
 	}
 
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			got := tt.logType.String()
-			if got != tt.want {
-				t.Errorf("not equals. want: %v, but got: %v", tt.want, got)
+	for _, test := range tests {
+		t.Run(test.name, func(tt *testing.T) {
+			defer goleak.VerifyNone(t)
```

Comment:
I don't know but ... is it correct to use `t` here?
i think hlts said `tt` is the correct one.

Reply:
Thank you for your review. 
Sorry. There was a correction omission:pray: